terraform/backup/destination/env/dev.tfvars

```diff
@@ -0,0 +1,2 @@
+source_account_id          = "393416225559"
+source_account_environment = "dev"
```

Comment:
Is it possible to name these `development` and `production`, I'm thinking just to keep it in sync with our normal environments.